packages/webapp/components/FooterNavBar.tsx

```diff
@@ -1,326 +1,104 @@
-  requiresLogin?: boolean;
-  shouldShowLogin?: boolean;
-  onClick?: () => void;
-  component?: (onClick: () => void) => ReactElement;
-};
-
-const notificationsPath = '/notifications';
-
-export const tabs: Tab[] = [
-  {
-    path: '/',
-    title: 'Home',
-    icon: (active: boolean) => (
-      <HomeIcon secondary={active} size={IconSize.Medium} />
-    ),
-  },
-  {
-    path: '/bookmarks',
-    title: 'Bookmarks',
-    icon: (active: boolean) => (
-      <BookmarkIcon secondary={active} size={IconSize.Medium} />
-    ),
-    shouldShowLogin: true,
-  },
-  {
-    path: '/search',
-    title: 'Search',
-    icon: (active: boolean) => (
-      <SearchIcon secondary={active} size={IconSize.Medium} />
-    ),
-  },
-  {
-    requiresLogin: true,
-    shouldShowLogin: true,
-    path: notificationsPath,
-    title: 'Notifications',
-    icon: (active: boolean, unreadCount) => (
-      <span className="relative">
-        {unreadCount > 0 ? (
-          <Bubble className="right-0 top-0 translate-x-1/2 px-1">
-            {getUnreadText(unreadCount)}
-          </Bubble>
-        ) : null}
-        <BellIcon secondary={active} size={IconSize.Medium} />
-      </span>
-    ),
-  },
-  {
-    path: '/filters',
-    title: 'Filters',
-    icon: (active: boolean) => (
-      <FilterIcon secondary={active} size={IconSize.Medium} />
-    ),
-  },
-];
-
-export const mobileUxTabs: Tab[] = [
-  {
-    path: '/',
-    title: 'Home',
-    icon: (active: boolean) => (
-      <HomeIcon secondary={active} size={IconSize.Medium} />
-    ),
-  },
-  {
-    path: '/search',
-    title: 'Search',
-    icon: (active: boolean) => (
-      <AiIcon secondary={active} size={IconSize.Medium} />
-    ),
-    shouldShowLogin: true,
-  },
-  {
-    component: (onClick: () => void) => (
-      <div key="new-post" className="text-center">
-        <CreatePostButton onClick={onClick} compact sidebar className="h-8" />
-      </div>
-    ),
-  },
-  {
-    requiresLogin: true,
-    shouldShowLogin: true,
-    path: '/squads',
-    title: 'Squads',
-    icon: (active: boolean) => (
-      <SourceIcon secondary={active} size={IconSize.Medium} />
-    ),
-  },
-  {
-    requiresLogin: true,
-    shouldShowLogin: true,
-    path: (user) => user?.permalink,
-    title: 'Profile',
-    icon: (active: boolean) => (
-      <UserIcon secondary={active} size={IconSize.Medium} />
-    ),
-  },
-];
+import { FooterTab } from './footer/common';
+import { FooterNavBarV1, mobileUxTabs } from './footer/FooterNavBarV1';
+import { FooterNavBarControl, tabs } from './footer/FooterNavBarControl';
 
 interface FooterNavBarProps {
   showNav?: boolean;
   post?: Post;
 }
 
+const selectedMapToTitle: Record<keyof UseActiveNav, string> = {
+  home: 'Home',
+  search: 'Search',
+  bookmarks: 'Bookmarks',
+  notifications: 'Notifications',
+  squads: 'Squads',
+  profile: 'Profile',
+};
+
+const combinedTabs = tabs.concat(
+  ...(mobileUxTabs.filter((tab) => !isValidElement(tab)) as FooterTab[]),
```

Comment:
why is this needed here?

Reply:
This is if in case there is no active page returned by the `useActiveNav`, we will check the paths here if any matches the current route.

Reply:
no, I mean why do you need to combine tabs if we are splitting the components rendering them?

Reply:
We have to support it the way we define which tab should be active for indicator using the `flipper`. Otherwise, we could've done it directly from either component.